gitbootcamp.rst

```diff
@@ -205,51 +201,48 @@ Commit the files:
 
 .. code-block:: bash
 
-   git commit -m '<message>'
-
+   git commit -m <message>
```

Comment:
I think this that is better used ```' '``` because when you write a message to the commit you need use ```' '```

Reply:
Oops, I was the one who initially suggested to remove the quotations as it initially looked like they were a styling decision. I forgot they were required.

Reply:
Okay, but let's use the double quotes as they allow the apostrophe punctuation mark in the commit message, and as they were already used in the two other `git commit` commands of this document. Done.